src/main/java/lotto/Lotto.java

```diff
@@ -8,20 +8,21 @@
 import java.util.stream.IntStream;
 
 public class Lotto {
-	private List<Integer> numbers;
+	private List<LottoNumber> numbers;
+
+	private final List<LottoNumber> allNumbers =
+		IntStream.range(START_NUMBER, END_NUMBER).mapToObj(LottoNumber::new).collect(Collectors.toList());
```

Comment:
체이닝 메서드의 경우 닷(.)을 한줄에 한개씩만 작성하는 방법도 좋을 것 같습니다.
가독성 측면 :)

```suggestion
	IntStream.range(START_NUMBER, END_NUMBER)
		.mapToObj(LottoNumber::new)
		.collect(Collectors.toList());
```

다른곳도 확인 부탁드려요~